packages/EasyErrorHandler/src/Bridge/Bugsnag/BugsnagReporter.php

```diff
@@ -13,35 +14,33 @@
 
 final class BugsnagReporter extends AbstractErrorReporter
 {
-    /**
-     * @var \Bugsnag\Client
-     */
-    private $bugsnag;
+    private Client $bugsnag;
 
     /**
      * @var string[]
      */
-    private $ignoreExceptions;
+    private ?array $ignoreExceptions;
```

Comment:
This cannot be null as we default it in the constructor.

Reply:
Fixed